go/vt/vttablet/tabletserver/vstreamer/helper_event_test.go

```diff
@@ -318,8 +319,11 @@ func (ts *TestSpec) Init() {
 
 // Close() should be called (via defer) at the end of the test to clean up the tables created in the test.
 func (ts *TestSpec) Close() {
-	dropStatement := fmt.Sprintf("drop tables %s", strings.Join(ts.schema.TableNames(), ", "))
-	execStatement(ts.t, dropStatement)
+	for _, table := range ts.schema.TableNames() {
+		dropStatement := fmt.Sprintf("drop table %s", table)
+		// Ignore the error as the table may not exist.
+		_ = env.Mysqld.ExecuteSuperQuery(context.Background(), dropStatement)
```

Comment:
I'm not sure that we can care much here, but we could also build a string with each individual drop statement and send that once to MySQL (a multi-statement query). That way we don't really pay a perf cost with this change.